src/state/buildBoard.js

```diff
@@ -26,31 +26,83 @@ export default function buildBoard(gameType = "classic") {
       board[0][4] = new PieceBoss({ type: ANUBIS, player: PLAYER_ONE });
       board[0][5] = new PieceBoss({ type: PHARAOH, player: PLAYER_ONE });
       board[0][6] = new PieceBoss({ type: ANUBIS, player: PLAYER_ONE });
-      board[0][7] = new PieceBoss({ type: PYRAMID, player: PLAYER_ONE });
-
-      board[1][2] = new PieceBoss({ type: PYRAMID, player: PLAYER_ONE });
-
-      board[2][3] = new PieceBoss({ type: PYRAMID, player: PLAYER_TWO });
+      board[0][7] = new PieceBoss({
+        type: PYRAMID,
+        player: PLAYER_ONE,
+        angle: 90,
```

Comment:
Would it make sense to abstract this to `direction`? Seems like we'd only ever want it to be one of four things and as angle it's not immediately obvious if it's rotated CW or CCW and which direction that ultimately leaves it in.

```
      board[0][7] = new PieceBoss({
        type: PYRAMID,
        player: PLAYER_ONE,
        direction: DIRECTIONS.DOWN,
      });
```

Reply:
Ahhh, that's a great point.

I think we have to decide what is up and down (facing which way). For now... maybe depending on what player you are it always orients you as the bottom? Could work that way for multiplayer. I think that's a great idea. It's definitely only multiples of 90 and they should be abstracted for ease of understanding.

I'll make that change.

Reply:
I did it but... I'm still not sold... what is "up"? How would I know? All the rotational pieces are angled at 45 degrees....

Food for thought.